File: 1353-find-resultant-array-after-removing-anagrams/1353-find-resultant-array-after-removing-anagrams.cpp
class Solution {
public:
    vector<string> removeAnagrams(vector<string>& words) {
        vector<string> ans;
        string prev="";
        for(string s:words){
            string srt=s;
            sort(srt.begin(),srt.end());
            if(srt!=prev){
                ans.push_back(s);
                prev=srt;
            }
        }
        return ans;
    }
};

File: 1353-find-resultant-array-after-removing-anagrams/1353-find-resultant-array-after-removing-anagrams.java
class Solution {
    public List<String> removeAnagrams(String[] words) {
        ArrayList<String> ans=new ArrayList<>();
        String prev="";
        for(String s:words){
            char[] c=s.toCharArray();
            Arrays.sort(c);
            String str=new String(c);
            if(!str.equals(prev)){
                ans.add(s);
                prev=str;
            }
        }
        return ans;
    }
}